types.ts

```diff
@@ -479,6 +479,20 @@ export interface CustomAuthParameter {
   description: string;
 }
 
+/**
+* An optional header hext digest transform for the {@link CustomAuthentication} authentication type.
```

Comment:
Nit, hext-->hex. Should probably include example usage and effect in the doc comment too.